eras/conway/impl/src/Cardano/Ledger/Conway/Rules/GovCert.hs

```diff
@@ -159,9 +194,17 @@ conwayGovCertTransition = do
     ConwayResignCommitteeColdKey coldK -> do
       checkColdKeyHasNotResigned coldK vsCommitteeHotKeys
       pure $ vState {vsCommitteeHotKeys = Map.insert coldK Nothing vsCommitteeHotKeys}
-    ConwayUpdateDRep cred _mAnchor -> do
+    ConwayUpdateDRep cred mAnchor -> do
       Map.notMember cred vsDReps ?! ConwayDRepNotRegistered cred
-      pure vState -- TODO: update anchor
+      expiryEpoch <-
+        calcDRepExpiryEpoch
+          cgceCurrentSlot
+          (cgcePParams ^. ppDRepActivityL)
```

Comment:
This has been repeated once here after seeing it above. May be we can turn it into a top-level function, since it needs to be used in CERTS as well later, if I remember correctly from the spec.

Reply:
Oh wait, it is already made into a function! Sorry!